script/client/saga/db/postgresql.sql

```diff
@@ -0,0 +1,59 @@
+-- -------------------------------- The script used for sage  --------------------------------
+CREATE TABLE IF NOT EXISTS public.seata_state_machine_def
+(
+    id               VARCHAR(32)  NOT NULL,
+    name             VARCHAR(255) NOT NULL,
+    tenant_id        VARCHAR(32)  NOT NULL,
+    app_name         VARCHAR(32)  NOT NULL,
+    type             VARCHAR(20),
+    comment_         VARCHAR(255),
+    ver              VARCHAR(16)  NOT NULL,
+    gmt_create       TIMESTAMP(0) NOT NULL,
+    status           VARCHAR(2)   NOT NULL,
+    content          TEXT,
+    recover_strategy VARCHAR(16),
+    CONSTRAINT pk_seata_state_machine_def PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS public.seata_state_machine_inst
+(
+    id                  VARCHAR(46)                NOT NULL,
+    machine_id          VARCHAR(32)                NOT NULL,
+    tenant_id           VARCHAR(32)                NOT NULL,
+    parent_id           VARCHAR(46),
+    gmt_started         TIMESTAMP(0)               NOT NULL,
+    business_key        VARCHAR(48),
+    start_params        TEXT,
+    gmt_end             TIMESTAMP(0) DEFAULT now(),
+    excep BYTEA,
+    end_params          TEXT,
+    status              VARCHAR(2),
+    compensation_status VARCHAR(2),
+    is_running          SMALLINT,
```

Comment:
test case failed: column "is_running" is of type smallint but expression is of type boolean